src/Microsoft.PowerShell.Commands.Utility/commands/utility/GetUnique.cs

```diff
@@ -89,9 +96,9 @@ protected override void ProcessRecord()
             else // compare as objects
             {
                 _comparer ??= new ObjectCommandComparer(
-                    true, // ascending (doesn't matter)
+                    ascending: true, // ascending (doesn't matter)
                     CultureInfo.CurrentCulture,
-                    true); // case-sensitive
+                    caseSensitive: !CaseInsensitive.IsPresent); // case-sensitive
```

Comment:
Given these use named arguments now maybe we can remove the comments as well?

Reply:
Yes, please remove the old comments.